src/ts/components/Match.tsx

```diff
@@ -48,19 +67,33 @@ class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
           </div>
           {suggestionContent}
           <div className="MatchWidget__annotation">{message}</div>
-          {this.props.feedbackHref && (
-            <div className="MatchWidget__feedbackLink">
-              <a
-                target="_blank"
-                href={this.getFeedbackLink(
-                  this.props.feedbackHref!,
-                  feedbackInfo
-                )}
-              >
-                Issue with this result? Tell us!
-              </a>
+          <div className="MatchWidget__footer">
+            {this.props.feedbackHref && (
+              <div className="MatchWidget__feedbackLink">
+                <a
+                  target="_blank"
+                  href={this.getFeedbackLink(
+                    this.props.feedbackHref!,
+                    feedbackInfo
+                  )}
+                >
+                  Issue with this result? Tell us!
+                </a>
+              </div>
+            )}
+            <div className="MatchWidget__ignore-match">
```

Comment:
The _actual_ tooltip and button markup!

Reply:
Surely this shouldn't be displayed unless onIgnoreMatch is defined?

Reply:
You're right! I've just made that change.